pkg/roles/roles.go

```diff
@@ -144,6 +145,8 @@ var (
 					"clustervolumeclasses",
 					"projectcomputeclasses",
 					"clustercomputeclasses",
+					"imageallowrules",
+					"clusterimageallowrules",
```

Comment:
Do `projectimageallowrules` need to be added here too?

Reply:
Absolutely!